### DISCUSSION TITLE
Is `axum_extra::response::Html` different from `axum::response::Html`?

### DISCUSSION THREAD

**Discussion by wt** [tokio-rs/axum#2630] [Q&A] [answered]
### Summary

These look extremely similar. Are these actually similar enough that the implementation in extras could be deprecated in favor of the one in the axum crate?

If not, could the differences be documented so that one know when to use one over the other?

### versions

* axum: 0.7.4
* axum-extra: 0.9.2

**Comment by jplatte** [comment]
The documentation has the difference, it's utf-8 vs. no specified encoding. I'm not sure why, though..

**Reply by jplatte** [reply, marked as answer]
Figured it out: The one in `axum-extra` is newer and was meant to be moved to `axum` in 0.7.0, but was forgotten. There's now an issue for moving it here: https://github.com/tokio-rs/axum/issues/2632

**Reply by wt** [reply]
Thanks.
